kinoml/core/proteins.py

```diff
@@ -137,18 +139,21 @@ def from_hgnc(cls, identifier, **kwargs):
     def from_name(cls, identifier, **kwargs):
         import requests
         import tempfile
-        from MDAnalysis import Universe
+        import MDAnalysis as mda
         from pathlib import Path
+        from appdirs import user_cache_dir
+
+        p = f"{user_cache_dir()}/{identifier}.pdb"
```

Comment:
* Use `path` instead of `p`
* Use `kinoml.utils.APPDIR` instead of the untagged call to `user_cache_dir`: https://github.com/openkinome/kinoml/blob/api/kinoml/utils.py#L8

Reply:
Example: https://github.com/openkinome/kinoml/blob/api/kinoml/datasets/kinomescan/utils.py#L43

Reply:
Done. Does this ensure a `kinoml` directory is created in `Cache`? If not do we need a check to make sure one is created?